src/java.base/share/classes/java/nio/file/CopyMoveHelper.java

```diff
@@ -110,13 +110,20 @@ static void copyToForeignTarget(Path source, Path target,
             Files.getFileAttributeView(source, PosixFileAttributeView.class);
 
         // attributes of source file
-        BasicFileAttributes sourceAttrs = sourcePosixView != null ?
-            Files.readAttributes(source,
-                                 PosixFileAttributes.class,
-                                 linkOptions) :
-            Files.readAttributes(source,
-                                 BasicFileAttributes.class,
-                                 linkOptions);
+        BasicFileAttributes sourceAttrs = null;
+        if (sourcePosixView != null) {
+            try {
+                sourceAttrs = Files.readAttributes(source,
+                                                   PosixFileAttributes.class,
+                                                   linkOptions);
+            } catch (SecurityException ignored) {
```

Comment:
Ignoring SecurityException looks right here but you need to do the same with targetPosixView.setPermissions(...) for the case that you are copying from a custom file system that supports permissions to the default file system.

Reply:
Fixed by 80961915b339fb3bf39acf120e25c363aff16afc.

Reply:
Would it be possible to add the comment
```
// okay to continue if RuntimePermission("accessUserInformation") not granted
```
in the catch block of both places so that it's clear to readers why it is okay to ignore the SE?